web/packages/design/src/SVGIcon/index.ts

```diff
@@ -69,3 +69,6 @@ export { UpgradeIcon } from './Upgrade';
 export { UserIcon } from './User';
 export { UsersIcon } from './Users';
 export { UserSettingsIcon } from './UserSettings';
+
+export { SVGIcon } from './SVGIcon';
```

Comment:
I'm getting confused.

I know we want to use phosphor icons now. ✅ 

But we also have svg icons that are not part of phosphor icons. I looked at storybook for our icons and we have like duplicates. for example "users", the phosphor icon and... "non-phosphor" icon looks different?

Reply:
i guess what i'm getting at is, should we clean it up and have phoshpor icon's be the source of truth?

Reply:
Yes, in a subsequent PR I'll clean up `SVGIcon` and remove duplicates, I didn't want to make this PR and larger than it is and wanted to keep its scope solely to replacing icomoon with Phosphor